### DISCUSSION TITLE
Secure - NoVPN video upload service.

### DISCUSSION THREAD

**Discussion by BarayeAzadi** [WeAreMahsaAmini/Core#355] [Ideas]
What if we create a service that 
- you send a message or email to,
- it will reply you with a URL.
- you can use the link to upload video, content or whatever you have recorded,
- we will have a series of automatic and manual verification mechanism to prove the authenticity and safety of the video.
- once approved, it will be shared either on the website or social media platform.

This way, sharing the video will be both anonymous and does not require a VPN service.

**Comment by 0xland** [comment]
Sounds good, but what will be the adaptation rate?

**Reply by BarayeAzadi** [reply]
if we share it with the right channel, it could become easily the only tool.
